STL_Extension/include/CGAL/Handle_for.h

```diff
@@ -36,6 +35,19 @@
 #  pragma warning(push)
 #  pragma warning(disable:4345) // Avoid warning  http://msdn.microsoft.com/en-us/library/wewb47ee(VS.80).aspx
 #endif
+
+#ifdef CGAL_HAS_THREADS
+#  define CGAL_HANDLE_FOR_USE_ATOMIC 1
+#endif // CGAL_HAS_THREADS
```

Comment:
It would be nice to protect CGAL_HANDLE_FOR_USE_ATOMIC with something a little less basic than CGAL_HAS_THREADS (especially when you see the benchmarks). There are applications that do not share CGAL objects between threads and are working fine today, they should have a way (possibly obscure and expert-only) to opt out of this slowdown; this would also be useful for testing and benchmarking.